workers/unity/Packages/com.improbable.gdk.core/Worker/WorkerConnector.cs

```diff
@@ -1,26 +1,38 @@
 using System;
 using System.Collections;
+using System.Collections.Generic;
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
-using Improbable.Gdk.Core;
 using Improbable.Worker.Core;
 using Unity.Entities;
 using UnityEngine;
 
 namespace Improbable.Gdk.Core
 {
-    public class WorkerConnectorBase : MonoBehaviour, IDisposable
+    public class WorkerConnector : MonoBehaviour, IDisposable
     {
         private delegate Task<Worker> ConnectionDelegate();
 
-        public GameObject LevelPrefab;
         public int MaxConnectionAttempts = 3;
         public bool UseExternalIp;
 
         public Worker Worker;
 
-        private GameObject levelInstance;
+        private List<Action<Worker>> workerConnectedCallbacks = new List<Action<Worker>>();
+
+        public event Action<Worker> OnWorkerCreationFinished
```

Comment:
This event doesn't appear to be used anywhere - is this in preparation for something else? Or was this an FR?

Reply:
Yes this was an FR. We want to use it to tell things like UI elements that stuff has happened.